Project-Days-Calendar/web.mjs

```diff
@@ -0,0 +1,11 @@
+// This is a placeholder file which shows how you can access functions and data defined in other files.
+// It can be loaded into index.html.
+// Note that when running locally, in order to open a web page which uses modules, you must serve the directory over HTTP e.g. with https://www.npmjs.com/package/http-server
+// You can't open the index.html file using a file:// URL.
+
+import { getGreeting } from "./common.mjs";
+import daysData from "../days.json" with { type: "json" };
+
+window.onload = function() {
+    document.getElementsByTagName("body")[0].innerText = `${getGreeting()} - there are ${daysData.length} know days`;
```

Comment:
```suggestion
    document.getElementsByTagName("body")[0].innerText = `${getGreeting()} - there are ${daysData.length} known days`;
```

(Separating in case you want to accept one but not the other 🙂)